### ISSUE TITLE
Smartphone display: popup not fully visible

### ISSUE THREAD

**Issue by jukos** [andreaordonselli/qgis2o.gis#12]
![popup outside](https://github.com/user-attachments/assets/1fcddbf1-2bf2-44b1-b54c-63284c9087b6)
In my test map the popup is not fully visible. In your demo map is all OK.
What could be the reason? (Other qgis2opengis version?)

Thank you and greetings

QGIS 3.36.3
qgis2opengis 2.21
Browser: Chrome

**Comment by andreaordonselli** [comment]
Strange, the popup is set centered in the smartphone view. Maybe you haven't updated the page?
Or is the width of your monitor greater than 650px despite being a smartphone? Retina screen?

![image](https://github.com/user-attachments/assets/26e0dfd8-4650-4bc6-9337-991ead149e9e)

**Comment by jukos** [comment]
Delete Browser data: yes
Data of my smartphone:
LTE-Smartphone
Android 12
6,67 Zoll Display
1.080 x 2.400 Pixel

**Comment by andreaordonselli** [comment]
This is the current code
```
////small screen definition
    var hasTouchScreen = map.getViewport().classList.contains('ol-touch');
    var isSmallScreen = window.innerWidth < 650;
```

When one of these two options is true the popup is centered.
I think it's better to have the popup centered in small screens, but tell me what you think.
in any case you can delete this piece of code and have the popup not centered

```
if (isSmallScreen && hasTouchScreen) {
			  popup.setPositioning('bottom-center')
			}
```

**Comment by jukos** [comment]
Which file?

**Comment by andreaordonselli** [comment]
Browse in resources/opengis/opengis.js
Soon in the new release
resources/o.gis/o.gis.js

**Comment by jukos** [comment]
I have commented out the line:
// var isSmall = window.innerWidth < 700;

The better solution for me.

**Comment by jukos** [comment]
It is certainly a balancing act. I prefer the method where the popup adjusts automatically. If there are a few pop-ups on the side and the user has to pull the pop-up inwards every time to read it in full, that is an unnecessary effort. 
Although it is a different topic, it is still interesting: In the case of sales via store systems, it is said that with every additional click, some customers leave the website. This must be prevented.

**Comment by andreaordonselli** [comment]
This issue has been resolved in the v2.22.0 release
Thanks for your report
